smokeTest/gameOfThrones.cpp

```diff
@@ -0,0 +1,56 @@
+/* The intention of this code is to solve the Game Of Thrones challenge from hackerrank
+ * The challenge can be seen here -> https://www.hackerrank.com/challenges/game-of-thrones/problem?h_r=internal-search
+ * Author: Lucas Lovato
+ * Date: 08/04/2020
```

Comment:
You could use `2020-04-08` (always interpreted as YYYY-MM-DD) to avoid misinterpretations